LJMD_serial/include/prototypes.h

```diff
@@ -0,0 +1,37 @@
+#ifndef PROTOTYPES_H
+#define PROTOTYPES_H
+
+#include "structures.h"
+#include "constants.h"
+
+#include <ctype.h>
+#include <math.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+#include <sys/time.h>
+
+#ifdef __cplusplus
+extern "C"
```

Comment:
@neerukulhari  why did you put extern "C"? We don't have C++ files

Reply:
I thought we did in examples, so maybe we need in future